pennylane/ops/channel.py

```diff
@@ -365,6 +367,91 @@ def _kraus_matrices(cls, *params):
         return [K0, K1, K2, K3, K4]
 
 
+class PauliError(Channel):
+    r"""PauliError(operators, p, wirelist, wires)
+    Arbitrary number qubit, arbitrary Pauli operator error channel.
+
+    This channel is modelled by the following Kraus matrices:
+
+    .. math::
+        K_0 = \sqrt{1-p} * I
+
+    .. math::
+        K_1 = \sqrt{p} * (K_{w0} \otimes K_{w1} \otimes \dots K_{wn})
+
+    Where :math:`I` is the Identity,
+    and :math:`\otimes` denotes the Kronecker Product,
+    and :math:`K_{wi}` denotes the Kraus matrix corresponding to the operator acting on wire i,
+    and :math:`p` denotes the probability with which the channel is applied
+
+    .. warning::
+
+        The size of the Kraus Matrices for PauliError scale exponentially
+        with the number of wires, the channel acts on. Simulations with
+        PauliError can result in a significant increase in memory and
+        computational usage. Use with caution!
+
+    **Details:**
+
+    * Number of wires: Any (the operation can act on any number of wires)
+    * Number of parameters: 3
+
+    Args:
+        operators (str): The Pauli operators acting on the specified (groups of) wires
+        p (float): The probability of the operator being applied
+        wirelist (Sequence[int]): The wires the channel acts on
```

Comment:
Don't we need wirelist as extra argument because we can't access wires?

Reply:
I think we could get away with only having `wires`, is that not the case? :thinking:

Reply:
if there is any way of accessing wires from the `kraus_matrices` function, then yes

Reply:
I see a way, yes. Since we only check if `wires` and `operators` have same dimensions in the `init` function, we can use the dimension of `operators` in `kraus_matrices`. Because as I remember, there was no way of accessing wires in `kraus_matrices`